pkg/test/BUILD

```diff
@@ -28,4 +28,5 @@ go_test(
         "//vendor/github.com/onsi/ginkgo:go_default_library",
         "//vendor/github.com/onsi/gomega:go_default_library",
     ],
+    tags = ["e2e"],
```

Comment:
I am not very clear about what's the usage of this tag?

Reply:
@m1093782566, this is to classify the tests. we generally use wildcard while invoking tests. if we have classification, we can omit certain class of tests.

Reply:
But if we manually edit this file, I am afraid that auto-generated BUILD file will overwrite it. Am I missing something?

Reply:
Hi @m1093782566, I could not figure how to achieve this in short term. So far now we can base our test invocation on paths. Later we shall figure out how to exclude tests based on tags, if absolutely needed.

Reply:
> So far now we can base our test invocation on paths.

Agree.